coordinator/src/trade/mod.rs

```diff
@@ -372,24 +392,43 @@ impl TradeExecutor {
         // coordinator.
         let event_id = format!("{contract_symbol}{maturity_time}");
 
-        let (offer_collateral, accept_collateral, fee_config) = match trader_required_utxos {
-            TraderRequiredLiquidity::ForTradeCostAndTxFees => (
-                margin_coordinator + collateral_reserve_coordinator.to_sat(),
-                margin_trader + collateral_reserve_trader + order_matching_fee,
-                dlc::FeeConfig::EvenSplit,
-            ),
-            TraderRequiredLiquidity::None => (
-                margin_coordinator
-                    + collateral_reserve_coordinator.to_sat()
-                    + margin_trader
-                    + collateral_reserve_trader
-                    // If the trader doesn't bring their own UTXOs, including the order matching fee
-                    // is not strictly necessary, but it's simpler to do so.
-                    + order_matching_fee,
-                0,
-                dlc::FeeConfig::AllOffer,
-            ),
-        };
+        let (offer_collateral, accept_collateral, fee_config, collateral_reserve_trader) =
```

Comment:
🔧 I think the last element of the tuple is the same for both branches, so you can omit it.

Reply:
yes, thank you that was a left over from a previous change.